GameDataParser/Parsers/MapEntityParser.cs

```diff
@@ -100,386 +67,320 @@ protected override List<MapEntityMetadata> Parse()
-                        MapNpc thisNpc = new MapNpc(int.Parse(npcId), modelName, name, CoordS.Parse(npcPositionValue), CoordS.Parse(npcRotationValue));
-                        thisNpc.IsSpawnOnFieldCreate = true;
-                        metadata.Npcs.Add(thisNpc);
-                    }
-                    else if (modelName == "EventSpawnPointNPC")
-                    {
-                        string npcId = node.SelectSingleNode("property[@name='NpcList']")?.FirstChild?.Attributes["index"].Value ?? "0";
-                        if (npcId == "0")
+                        break;
+                    case IMS2RegionSpawnBase spawnBase:
+                        switch (spawnBase)
                         {
-                            continue;
+                            case IMS2RegionBoxSpawn boxSpawn:
+                                SpawnMetadata mobSpawnDataBox =
+                                    (SpawnTagMap.ContainsKey(mapId) && SpawnTagMap[mapId].ContainsKey(boxSpawn.SpawnPointID))
+                                        ? SpawnTagMap[mapId][boxSpawn.SpawnPointID]
+                                        : null;
+
+                                int mobNpcCountBox = mobSpawnDataBox?.Population ?? 6;
+                                int mobSpawnRadiusBox = 150;
+                                // TODO: This previously relied in "NpcList" to be set. NpcList is impossible to be set on
+                                // MS2RegionSpawn, it's only set for SpawnPointNPC.
+                                List<int> mobNpcListBox = new List<int>();
+                                mobNpcListBox.Add(21000025); // Placeholder
+                                metadata.MobSpawns.Add(new MapMobSpawn(boxSpawn.SpawnPointID, ToCoordS(boxSpawn.Position),
+                                    mobNpcCountBox, mobNpcListBox, mobSpawnRadiusBox, mobSpawnDataBox));
+                                // "QR_10000264_" is Quest Reward Chest? This is tied to a MS2TriggerAgent making this object appear.
+                                break;
+                            case IMS2RegionSpawn regionSpawn:
+                                SpawnMetadata mobSpawnData =
+                                    (SpawnTagMap.ContainsKey(mapId) && SpawnTagMap[mapId].ContainsKey(regionSpawn.SpawnPointID))
+                                        ? SpawnTagMap[mapId][regionSpawn.SpawnPointID]
+                                        : null;
+
+                                int mobNpcCount = mobSpawnData?.Population ?? 6;
+
+                                // TODO: This previously relied in "NpcList" to be set. NpcList is impossible to be set on
+                                // MS2RegionSpawn, it's only set for SpawnPointNPC.
+                                List<int> mobNpcList = new List<int>();
+                                mobNpcList.Add(21000025); // Placeholder
```

Comment:
Same thing for line 241

```suggestion
                                List<int> mobNpcListBox = new List<int>
                                {
                                    21000025 // Placeholder
                                };
```